### PR TITLE
add codecov

### PR DESCRIPTION
**Pull request by stevenysw**
I am an rOpensci intern, working on adding test coverage components.

### CODE REVIEW

File: tests/testthat/test-lexical_sentences.R [hunk 1]
```diff
@@ -0,0 +1,8 @@
+context("lexical_sentences")
+
+test-that("lexical_sentences", {
```
Comment (sckott) [inline tests/testthat/test-lexical_sentences.R]: @steven2249 this should be `test_that` not `test-that`

File: tests/testthat/test-lexical_pmentities.R [hunk 2]
```diff
@@ -0,0 +1,13 @@
+context("lexical_pmentities")
+
+test_that("lexical_pmentities", {
+  tt <- lexical_pmentities(pmid = 1)
+  
+  expect_is(tt, "data.frame")
+  expect_is(tt$token, "character")
+  expect_equal(length(tt$token), 4)
+  expect_is(tt$start, "integer")
+  expect_equal(tt$start, c(0, 7, 45, 66))
```
Comment (sckott) [inline tests/testthat/test-lexical_pmentities.R]: @steven2249 running locally, the test on line 10 fails for me. check that one - I'm guessing that the value returned can vary through time - if so, remove that test, or change it to something that will pass no matter when it's called.

**Review by sckott — CHANGES_REQUESTED** [review]
nice work @steven2249 !

just a few things to change and then we're all good

File: tests/testthat/test-lexical_pmentities.R (hunk 2 above)
Comment (stevenysw) [reply tests/testthat/test-lexical_pmentities.R]: removed

File: tests/testthat/test-lexical_sentences.R (hunk 1 above)
Comment (stevenysw) [reply tests/testthat/test-lexical_sentences.R]: revised

**Review by sckott — APPROVED** [review]

**Comment by sckott** [comment]
thanks @steven2249 !

this all looks good

**Pull request merged**